lib/SparkPost/Transmission.php

```diff
@@ -89,6 +89,13 @@ class Transmission extends APIResource {
    * @return array API repsonse represented as key-value pairs
    */
   public function send( $transmissionConfig ) {
+    if(isset($transmissionConfig["startTime"]) &&
+      $transmissionConfig["startTime"] instanceof \DateTime)
+      {
+        $transmissionConfig["startTime"] =
+          $transmissionConfig["startTime"]->format(\DateTime::ATOM);
```

Comment:
I was trying to keep ahead of the PSR-2 move (lines should be under 80 characters), but I have to admit it's more readable on one line, and 100 characters is fine these days :)